src/sys/signal.rs

```diff
@@ -445,6 +445,27 @@ pub fn pthread_sigmask(how: SigmaskHow,
     Errno::result(res).map(drop)
 }
 
+/// Examine and change blocked signals.
+///
+/// For more informations see the [`sigprocmask` man
+/// pages](http://man7.org/linux/man-pages/man2/sigprocmask.2.html).
```

Comment:
This is a portable function, so you should link to a portable man page.
http://pubs.opengroup.org/onlinepubs/9699919799/functions/sigprocmask.html